src/main/java/net/nextencia/rrdiagram/Main.java

```diff
@@ -126,7 +126,7 @@ public String getLink(String ruleName) {
           pw.write("\n```\n");
```

Comment:
The only other triple backticks I see is on line 124 (2 lines above this) and README (should not change that).  Please add ebnf for the above one, even though it is currently in comments.

Reply:
Updated!